frontend/e2e/nightwatch-entrypoint.bash

```diff
@@ -5,7 +5,25 @@ function trapint {
     exit 0
 }
 
-echo 'Waiting for app to boot...'
+echo 'Waiting for backend to boot...'
```

Comment:
from Skylight, this should resolve race conditions between e2e tests and gradle bootRun :)